<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ustawienia</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>571</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Log</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nazwa logu:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="logTextEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="placeholderText">
     <string>AI.log</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="czyscPlikCheckBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Czyść przed nowym zapisem  </string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>571</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Sieć neuronowa</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Liczba neuronów ukrytej warstwy 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Liczba neuronów ukrytej warstwy 2:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="liczbaNeuronow1SpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="liczbaNeuronow2SpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wysokość:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="wysokoscSpinBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Szerokość:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="szerokoscSpinBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Limit pamięci dla learn chunk (w KB):</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="learnChunkSpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Ścieżki do danych</string>
   </property>
   <widget class="QPushButton" name="testDataButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="sciezkaDaneUczaceTextEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ścieżka do danych uczących:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trainingDataButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="sciezkaDaneTestoweTextEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ścieżka do katalogów z danymi testowymi:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Używane grupy (regex):</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="regexTextEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="placeholderText">
     <string>default.*</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
